### DISCUSSION TITLE
having more than 100 people connectet to a Hub // server increase?

### DISCUSSION THREAD

**Discussion by tiemposdelruido** [mozilla/hubs#2847] [General Questions & Discussion] [answered]
Hello, all! I am hosting in two weeks a concert in the Toplap Berlin Hub https://hubs.mozilla.com/oUQRigk/toplap-berlin with the Goethe Institut in Moskow. It is the opening of a conference. In my previous experiences with Hubs, when more than 25 people connect to the hub there are problems at loading the objects for everybody. I was told that there is a way to increase the server capacity, but I haven't found it. We are expecting more than 100 people to connect to our event. Could anyone suggest to me what's the best way to increase the capacity of the server and ensure a smooth concert? Thank you very much!

**Comment by johnshaughnessy** [comment, marked as answer]
Somewhere around 25 people is the most that one room can support. Hubs will fail before reaching 100 people connected to the same room. Other events I've seen have worked around this limitation by having multiple rooms with ways to navigate between them. (This could be as simple as a list of URLs, or you could setup clickable links from one room to another by pasting URL's into the rooms themselves and then "pinning" them there.)

We do not typically support large events on hubs.mozilla.com -  instead we direct people to use Hubs Cloud - https://hubs.mozilla.com/cloud Hubs cloud allows you to configure the server capacity depending on your needs, whereas large surges in traffic to hubs.mozilla.com may result in people not being able to connect to your event rooms. (Note that server capacity is different than room capacity. Room capacity is fixed at ~25. Server capacity accounts for having many rooms.)

More details here : https://hubs.mozilla.com/docs/hubs-faq.html#what-is-the-capacity-of-a-hubs-room

**Reply by tiemposdelruido** [reply]
Thank you @johnshaughnessy for your answer!

**Reply by InfiniteLee** [reply]
25 is only a "limit" that we've observed on specific devices like iOS. https://hubs.mozilla.com is currently configured with a hard cap of 50 in-room (no cap on in-lobby), but that number is also adjustable on a Hubs Cloud server.

Also see: https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-ccu-limits.html with regards to how to accommodate varying levels of ccu on Hubs Cloud.
